benchmarks/ner_embeddings/scripts/run_experiments.py

```diff
@@ -0,0 +1,398 @@
+
+app = typer.Typer()
+
+
+class StaticVectors(str, Enum):
+    spacy = "spacy"
+    fasttext = "fasttext"
+    null = "null"
+
+
+def _run_commands(cmds: List[str], dry_run: bool = False):
+    """Run a set of commands in order"""
+    for cmd in cmds:
+        _cmd = shlex.split(cmd)
+        if dry_run:
+            print(cmd.strip())
+        else:
+            subprocess.run(_cmd)
+
+
+def _make_train_command(
+    dataset: str,
+    config: str,
+    lang: str,
+    vectors: str,
+    gpu_id: int,
+    seed: int,
+    include_static_vectors: bool,
+    adjust_rows: bool = False,
+    adjust_value: int = 1,
+    tables_path: str = "tables",
+    batch_size: int = 1000,
+    num_hash: Optional[int] = None,
+) -> str:
+    """Construct train command based from a template"""
+    cmd_vectors = ""
+    cmd_rows = ""
+    cmd_hash = ""
+    modifier = ""
+
+    if not include_static_vectors:
+        cmd_vectors = "--vars.include_static_vectors false"
+    if adjust_rows and config != "ner_multiembed":
+        modifier = "-custom-rows"
+        new_rows = _get_computed_rows(tables_path, dataset, adjust_value)
+        cmd_rows = f"--vars.rows '{new_rows}'"
+    if num_hash:
+        modifier = "-hash"
+        cmd_hash = f"--vars.num_hashes {num_hash}"
+
+    command = (
+        f"spacy project run train{modifier} .  "
+        f"--vars.dataset {dataset} "
+        f"--vars.ner_config {config} "
+        f"--vars.language {lang} "
+        f"--vars.vectors {vectors} "
+        f"--vars.gpu-id {gpu_id} "
+        f"--vars.seed {seed} "
+        f"--vars.tables_path {tables_path} "
+        f"--vars.batch_size {batch_size} "
+        f"{cmd_vectors} {cmd_rows} {cmd_hash}"
+    )
+    return command
+
+
+def _format_attrs(attrs: List[str]) -> str:
+    s = ""
+    for idx, attr in enumerate(attrs):
+        s += f'"{attr}"'
+        if idx != len(attrs) - 1:
+            s += ", "
+    cmd_attrs = f"{s}"
+    return cmd_attrs
+
+
+def _get_computed_rows(tables_path: str, dataset: str, adjust_value: int) -> List:
+    attrs = ["NORM", "PREFIX", "SUFFIX", "SHAPE"]
+    table_filepath = f"{tables_path}/{dataset}/{dataset}-train.tables"
+    table = srsly.read_msgpack(table_filepath)
+    attr_sizes = {attr: int(len(table[attr]) / adjust_value) for attr in attrs}
+    msg.text(f"Adjusted to {attr_sizes}")
+    return list(attr_sizes.values())
+
+
+def _make_eval_command(
+    dataset: str,
+    config: str,
+    gpu_id: int,
+    vectors: str,
+    seed: int,
+    metrics_dir: str,
+    eval_unseen: bool = False,
+) -> str:
+    """Construct eval command based from a template"""
+    unseen_param = "-seen-unseen" if eval_unseen else ""
+    command = f"""
+    spacy project run evaluate{unseen_param} .
+    --vars.ner_config {config}
+    --vars.dataset {dataset}
+    --vars.gpu-id {gpu_id}
+    --vars.vectors {vectors}
+    --vars.seed {seed}
+    --vars.metrics_dir {metrics_dir}
+    """
+    return command
+
+
+def _make_hash_command(
```

Comment:
maybe more descriptive?
```suggestion
def _make_tables_command(
```